doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -530,52 +626,18 @@ get_state_info(self):
 ```
 
 #### DPU_HEALTH Use Case
-* The major consumer of this data could be CLIs, fault management, debug, error recovery 
-
-get_health_info(self):
-```
-    Retrieves the dpu health object having the detailed dpu health Fetched from the DPUs
-
-    Returns:
-        An object instance of the dpu health. 
-        Returns None when the module is SWITCH
-    
-    Example:
-    {
-      "led_status": "green",
-
-      "monitoredlists": {
-          "Program": [
-              {"Name": "routeCheck", "Status": "Not OK", "Type": "Program"},
-              // Add more program items here
-          ],
-          "Service": [
-              {"Name": "mgmt-framework", "Status": "Not OK", "Type": "Service"},
-              // Add more service items here
-          ],
-          "Fan": [
-              {"Name": "Fan", "Status": "Not OK", "Type": "Fan"}
-          ],
-          "UserDefined": [
-              // Add user-defined items here
-          ]
-      },
-
-      "ignore_list": [
-          {"Name": "example1", "Status": "OK", "Type": "Type1"},
-          // Add more items to ignore list
-      ]
-    }
-```
+* The major consumer of this data could be CLIs, fault management, debug, error recovery.  There is no platform API for this.
 
 ### 3.2 Thermal management
 * Platform  initializes all sensors
 * Thermalctld fetch CPU temperature, DPU temperature, fan speed, monitor and update the DB
+* The thermal management runs on NPU
 * Thermal manager reads all thermal sensor data, run thermal policy and take policy action Ex. Set fan speed, set alarm, set syslog, set LEDs 
 * Platform collects fan related data such as presence, failure and then applies fan algorithm to set the new fan speed
 * The north bound CLI/Utils/App use DB data to ”show environment”, ”show platform temp” show platform fan”
 * The DPUs will update the ChassisStateDB "TEMPERATURE_INFO" tables through redis client call which in turn will be pushed into the switch StateDB.
 * The existing "TEMPERATURE_INFO" schema will be used to store the values and is shown below for convenience.
+* For phase:1 implementation the sensor values collected by DPU will not be pushed to the chassisStateDB.
```

Comment:
Why? Won't be pushed by whom and on which platform?

Reply:
@oleksandrivantsiv cleaned it.

Reply:
#closed